pmda/rmsf.py

```diff
@@ -0,0 +1,158 @@
+            # debugging.
+            if scheduler is None and n_jobs == 1:
+                scheduler = 'single-threaded'
+
+            # fall back to multiprocessing, we tried everything
+            if scheduler is None:
+                scheduler = 'multiprocessing'
+
+            scheduler_kwargs = {'scheduler': scheduler}
+            if scheduler == 'multiprocessing':
+                scheduler_kwargs['num_workers'] = n_jobs
+
+            self._indices = self._atomgroup.indices
+            self._ind_slices = np.array_split(self._indices, n_blocks)
+
+            with timeit() as total:
+                with timeit() as prepare:
+                    self._prepare()
+            start, stop, step = self._trajectory.check_slice_indices(
+                start, stop, step)
+            self._frames = range(start, stop, step)
+            blocks = []
+            for ind_slice in self._ind_slices:
+                task = delayed(
+                    self._dask_helper, pure=False)(
+                             ind_slice,
+                             self._top,
+                             self._traj,
+                             self._frames)
+                blocks.append(task)
+            blocks = delayed(blocks)
+            res = blocks.compute(**scheduler_kwargs)
+            self._results = res
+            self.sumsquares = res[0][0]
+            self.mean = res[0][1]
+            with timeit() as conclude:
+                self._conclude()
+            # self.timing = Timing(times_io,
+            #                      np.hstack(timings), total.elapsed,
+            #                      b_universe.elapsed, prepare.elapsed,
+            #                      conclude.elapsed)
+            return self
+
+    def _conclude(self):
+        # Called once iteration on the trajectory is finished. Results
+        # for each frame are stored in ``self._results`` in a per block
+        # basis. Here those results should be moved and reshaped into a
+        # sensible new variable.
+        # Apply normalization and averaging to results here if wanted.
+        k = len(self._frames)
+        self.rmsf = np.sqrt(self.sumsquares.sum(axis=1) / k)
+
+
+    def _dask_helper(self, ind_slice, top, traj, frames):
+        """helper function to actually setup dask graph"""
+        # wait_end needs to be first line for accurate timing
+        wait_end = time.time()
+        with timeit() as b_universe:
+            u = mda.Universe(top, traj)
+            agroups = u.atoms[ind_slice]
+
+        sumsquares = np.zeros((agroups.n_atoms, 3))
+        mean = np.zeros((agroups.n_atoms, 3))
+        for i, frame in enumerate(frames):
+            with timeit() as b_io:
+                ts = u.trajectory[frame]
+                frame_index = i
+                sumsquares, mean = self._single_frame(ts, agroups, frame_index, sumsquares, mean)
+        return sumsquares, mean
```

Comment:
Missing the timing information in the return – you should not need to worry about this when using the base class `_dask_helper()`.